io.gitlab.librewolf-community.json

```diff
@@ -0,0 +1,84 @@
+    "modules": [
+        "shared-modules/dbus-glib/dbus-glib-0.110.json",
+        {
+            "name": "librewolf",
+            "buildsystem": "simple",
+            "only-arches": [
+                "x86_64",
+                "aarch64"
+            ],
+            "build-commands": [
+                "mkdir -p /app/lib",
+                "mv librewolf.tar.bz2 /app/lib/librewolf",
+                "install -D -m644 /app/lib/librewolf/browser/chrome/icons/default/default128.png /app/share/icons/hicolor/128x128/apps/io.gitlab.librewolf-community.png",
+                "install -D -m644 /app/lib/librewolf/browser/chrome/icons/default/default128.png /app/usr/share/icons/hicolor/128x128/apps/io.gitlab.librewolf-community.png",
+                "install -D -m644 /app/lib/librewolf/io.gitlab.librewolf-community.appdata.xml -t /app/share/metainfo",
+                "install -D -m644 io.gitlab.librewolf-community.desktop -t /app/share/applications",
+                "install -D -m755 librewolf.sh /app/bin/librewolf"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://gitlab.com/librewolf-community/browser/linux/-/jobs/1267046474/artifacts/raw/LibreWolf-88.0.1-2.aarch64.tar.bz2",
+                    "sha256": "660284b741f6d2507e5d1eb3d423ad96577f59b7902c3028d189742c8cdb7424",
+                    "dest": "librewolf.tar.bz2",
+                    "strip-components": 0,
+                    "only-arches": [
+                        "aarch64"
+                    ]
+                },
+                {
+                    "type": "archive",
+                    "url": "https://gitlab.com/librewolf-community/browser/linux/-/jobs/1265492102/artifacts/raw/LibreWolf-88.0.1-2.x86_64.tar.bz2",
+                    "sha256": "d16fc62c8cbce90854de8793523d6da946dacc195edfc4abef6452fcce05651e",
+                    "dest": "librewolf.tar.bz2",
+                    "strip-components": 0,
+                    "only-arches": [
+                        "x86_64"
+                    ]
+                },
+                {
+                    "type": "file",
+                    "path": "io.gitlab.librewolf-community.desktop",
+                    "dest-filename": "io.gitlab.librewolf-community.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "librewolf.sh",
+                    "dest-filename": "librewolf.sh"
+                }
+            ]
+        }
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--socket=pcsc",
+        "--socket=pulseaudio",
+        "--persist=.librewolf",
+        "--persist=.mozilla",
+        "--filesystem=xdg-download:rw",
+        "--filesystem=xdg-run/pipewire-0",
+        "--device=dri",
+        "--filesystem=~/.config/dconf:ro",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
```

Comment:
What do you need dconf access for?

Reply:
Also related to the comment above and https://github.com/flatpak/flatpak/issues/2861.

I'll try a build without it, maybe it's not an issue anymore so it could be removed as well ^^

Reply:
Seems to not be required anymore – removed.